app/models/owner.js

```diff
@@ -67,6 +69,36 @@ export default VcsEntity.extend({
     }, { live: false });
   },
 
+  fetchPlans: task(function* () {
+    const url = this.isOrganization ? `/plans_for/organization/${this.id}` : '/plans_for/user';
+    const result = yield this.api.get(url);
+    return result ? result.plans : [];
+  }).keepLatest(),
+
+  fetchPlansInstance: computed(function () {
+    return this.fetchPlans.perform();
+  }),
+
+  isFetchPlansRunning: reads('fetchPlansInstance.isRunning'),
+  eligiblePlans: reads('fetchPlansInstance.value'),
+
+  nonGithubPlans: computed('eligiblePlans.@each.{id,name,annual,builds}', function () {
+    const eligiblePlans = this.eligiblePlans || [];
+    return eligiblePlans.filter(plan => plan.id && !plan.id.startsWith('github'));
+  }),
+
+  monthlyPlans: computed('nonGithubPlans.@each.{name,annual,builds}', function () {
+    const nonGithubPlans = this.nonGithubPlans || [];
```

Comment:
`nonGithubPlans` should always be an array anyway, because it always returns the results of `filter`